chapters/annotations.tex

```diff
@@ -297,7 +290,7 @@ \section{Annotations for Graphical Objects}\label{annotations-for-graphical-obje
   annotation "(" [ layer-annotations ] ")"
 
 layer-annotations :
-  ( icon\_layer | diagram\_layer ) [ "," layer-annotations ]
+  ( icon_layer | diagram_layer ) [ "," layer-annotations ]
```

Comment:
It should `icon-layer` and `diagram-layer` instead, and was wrong before as well - this was missed when generally replacing underscore in all grammar constructs by dash.

Reply:
Fixed.  Resolving.